src/veins/modules/utility/Consts80211p.h

```diff
@@ -140,6 +140,16 @@ enum ChannelNumber {
     HPPS = 184
 };
 
+constexpr std::array<std::pair<int, double>, 7> Mac1609_4_Center_Frequencies = {
```

Comment:
breaks in C++11 (the `std::pair` constructor only became `constexpr` in C++14)

Reply:
Then we have to move this back to a compilation unit, i.e., Mac1609_4.
Another reason I am hoping for an upgrade to C++14 soon...